packages/@aws-cdk/cloudformation-include/test/valid-templates.test.ts

```diff
@@ -414,6 +414,67 @@ describe('CDK Include', () => {
       includeTestTemplate(stack, 'non-existent-resource-type.json');
     }).toThrow(/Unrecognized CloudFormation resource type: 'AWS::FakeService::DoesNotExist'/);
   });
+
+  test("can ingest a template that contains outputs and modify them", () => {
+    const cfnTemplate = includeTestTemplate(stack, 'outputs.json');
+    const output = cfnTemplate.getOutput('Output1');
+    const bucket = cfnTemplate.getResource('Bucket');
+
+    output.setValue('a mutated value');
+    output.setDescription(undefined);
+    output.setExport("an export");
+    output.setCondition(new core.CfnCondition(stack, 'MyCondition', {}));
+
+    new core.CfnOutput(stack, 'Output2', {
+      value: bucket.ref,
+      description: "a description",
+    });
+
+    const originalTemplate = loadTestFileToJsObject('outputs.json');
+
+    expect(stack).toMatchTemplate({
+      "Conditions": {
+        ...originalTemplate.Conditions,
+      },
+      "Outputs": {
+        "Output1": {
+          "Value": "a mutated value",
+          "Export": {
+            "Name": "an export",
+          },
+          "Condition": "MyCondition",
+        },
+        "Output2": {
+          "Value": {
+            "Ref": "Bucket",
+          },
+          "Description": "a description",
+        },
+      },
+      "Resources": {
```

Comment:
I think "Resources" should be in the assertion before "Outputs"... that's the usual order in the CloudFormation template.